cell_workers/pokemon_catch_worker.py

```diff
@@ -8,6 +9,7 @@ def __init__(self, pokemon, bot):
         self.position = bot.position
         self.config = bot.config
         self.pokemon_list = bot.pokemon_list
+        self.log = logging.getLogger(__name__)
```

Comment:
You can use the logger available at `bot.log`, there's no need to create the logger again here.